components/datadog/apps/dogstatsd/docker-compose.yaml

```diff
@@ -0,0 +1,22 @@
+version: "3.9"
+services:
+  dogstatsd-uds:
+    image: ubuntu
+    container_name: metric-sender-uds
+    network_mode: host
+    pid: "host"
+    volumes:
+      - /var/run/datadog:/var/run/datadog:ro
+    command: /bin/bash -ec 'apt update; apt install -y netcat; while true; do echo -n "custom.metric.uds:1|c" | nc -U -u -w1 /var/run/datadog/dsd.socket; sleep 10; done'
+    restart: "no" 
+    depends_on:
+      - agent
+  dogstats-udp:
```

Comment:
For some reason, the metrics send over UDP are not found on the DD backend.

The docker-compose definition of the app seems correct to me, I can't figure out if the issue is related to dogstatsd or to some missing configuration. 

`agent config` and `agent dogstatsd-stats` show that the metric is received over UDP, but still can't see it on the DD app.